README.md

```diff
@@ -126,10 +126,10 @@ You can use MongoDB either as the main database, either as a side database. To d
     'username' => env('DB_USERNAME', 'homestead'),
     'password' => env('DB_PASSWORD', 'secret'),
     'options' => [
-        'database' => 'admin', // required with Mongo 3+
-
-        // here you can pass more settings
-        // see https://www.php.net/manual/en/mongodb-driver-manager.construct.php under "Uri Options" for a list of complete parameters you can use
+        // here you can pass more settings to the Mongo Driver Manager
+        // see https://www.php.net/manual/en/mongodb-driver-manager.construct.php under "Uri Options" for a list of complete parameters that you can use
+        
+        'authSource' => env('DB_AUTHENTICATION_DATABASE', 'admin'), // required with Mongo 3+
```

Comment:
I never user `'authSource'`, I have always used `'database'.  Is this correct?

I am also using 
`'dump' => [
		        'mongodb_user_auth' => env('DB_AUTHENTICATION_DATABASE', 'admin') //Auth for laravel backup plugin
	        ]` 
for backup. But I do not know if it is a specific configuration of [spatie backup](https://docs.spatie.be/laravel-backup/v6/installation-and-setup/#dumping-the-database) and if it makes sense to mention it.

Reply:
I had the weird feeling that `database` is not quite what i was using before and somehow had to look up for the available settings.

The setting for the dump can be added by the users that use Spatie's Backup package.

Reply:
Ok for the dump option. It is ok for me. You can merge